mod/wet4/views/default/navigation/menu/elements/item.php

```diff
@@ -49,6 +55,14 @@
 	$roles = "role=\"$item_role\" ";
 }
 
+// CL 20221213 - Get the menu title, if matches Network or Réseau change the url to friend/$username instead of /members
+// CL 20221213 - re: https://github.com/gctools-outilsgc/gcconnex/issues/2474
+if($item->getText() == 'Network' || $item->getText() == "Réseau"){
```

Comment:
There's a less hacky way that this can be done in elgg even with how much of a mess the we4 mod is:
First, removing the members menu item, the mod already removes a lot so this fits right in:
`elgg_unregister_menu_item('site', 'members');`
with this bunch  https://github.com/gctools-outilsgc/gcconnex/blob/master/mod/wet4/start.php#L258


and the colleagues / friends menu item already gets registered further down in the same file, so that can just be adjusted to instead add it to the site menu instead over here:
https://github.com/gctools-outilsgc/gcconnex/blob/master/mod/wet4/start.php#L538